sourced/ml/__main__.py

```diff
@@ -80,15 +81,25 @@ def add_parser(name, help):
     # ------------------------------------------------------------------------
     repos2coocc_parser = add_parser(
         "repos2coocc", "Convert source code to the sparse co-occurrence matrix of identifiers.")
+    repos2coocc_parser.set_defaults(handler=repos2coocc_entry)
     add_engine_args(repos2coocc_parser)
     add_repo2_args(repos2coocc_parser, quant=False)
+    add_split_stem_arg(repos2coocc_parser)
     repos2coocc_parser.add_argument(
         "-o", "--output", required=True,
         help="[OUT] Path to the Cooccurrences model.")
-    repos2coocc_parser.add_argument(
-        "--split-stem", default=False, action="store_true",
-        help="Split Tokens to parts (ThisIs_token -> ['this', 'is', 'token']).")
-    repos2coocc_parser.set_defaults(handler=repos2coocc_entry)
+    # ------------------------------------------------------------------------
+    repos2roles_and_ids = add_parser(
+        "repos2roles_ids", "Converts a UAST to a list of pairs, where pair is a role and "
```

Comment:
list of pairs of identifiers and corresponding UAST node roles.

BTW: internal role or merged generic roles?

Reply:
Merged generic roles.

Reply:
Include to the docstring